cartridges/app_custom_account_verification/cartridge/controllers/Account.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+/**
+ * @namespace Account
+ */
+
+var server = require('server');
+
+var base = server.extend(module.superModule);
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+var userLoggedIn = require('*/cartridge/scripts/middleware/userLoggedIn');
+var consentTracking = require('*/cartridge/scripts/middleware/consentTracking');
+var Encoding = require('dw/crypto/Encoding');
+
+function decodeId(encodedId) {
```

Comment:
You have encode function in the accountHelper already. Although both should be in a utilityHelper as these are not related to the account